src/main/java/teammates/ui/controller/StudentFeedbackResultsPageData.java

```diff
@@ -171,16 +174,24 @@ private FeedbackResultsResponseTable createResponseTable(FeedbackQuestionAttribu
                       !(recipientName.startsWith("Your Team (") && recipientName.endsWith(")"))) { // To avoid duplicate replacement
```

Comment:
The logic for changing the display name to 'You' or 'Your team' seems to be quite messy here, with conditions to avoid duplicate replacement... Do refactor if possible.